binutils/plan.sh

```diff
@@ -27,9 +28,7 @@ do_prepare() {
   export CFLAGS="$CFLAGS -static-libgcc"
   build_line "Updating CFLAGS=$CFLAGS"
 
-  # TODO: For the wrapper scripts to function correctly, we need the full
```

Comment:
We made it *super* far without ever going back to fix this TODO up. I can't believe it.